Updated input generation script

anastasia-shch opened the pull request:
- added config template to be populated with newly generated data and used by the integration tests
- fixed get_chip_from_anno.sh to sync batch names in the prepared basic info and chip table (db)
- updated integration tests

RigbeGW reviewed (approved):
I guess this is removing the previous commit that added a logic to skip the step if the files existed and as far as i can tell looks ok

RigbeGW commented on scripts/README.md:
looks good

RigbeGW commented on scripts/prepare_genotypeB_input.sh:
I guess this is undoing the previous commit of the logic to skip files if they exist, looks ok to me

RigbeGW commented on server/gt_test.py:
I didnt test this but am sure you have, but one question I have regarding this is the try block starting at line 84 is handling different assertions (not just one) and I guess would be difficult to tell which part is raising the exception so may be would be better to refactor?

anastasia-shch committed:
- Added config template to be populated with newly generated data and used by the integration tests
- Fixed get_chip_from_anno.sh to sync batch names in the prepared basic info and chip table (db)
- Updated integration tests
- Added R12 inputs for preparing GB inputs

anastasia-shch committed:
Updated to the latest main to solve conflicts.
Merge branch 'main' into feature_update_inputgen_script_v1

anastasia-shch commented:
@RigbeGW I squashed all commits into one to cleanup history and refactored integration tests as recommended.
